xgcm/test/test_grid.py

```diff
@@ -143,7 +143,7 @@ def test_create_axis_no_coords(all_datasets):
     ds, periodic, expected = all_datasets
     axis_objs = _get_axes(ds)
 
-    ds_drop = ds.drop(list(ds.coords))
+    ds_drop = ds.drop_vars(list(ds.coords))
```

Comment:
Interesting. What is the difference here?

Reply:
Just more explicit.

Reply:
Awesome, thanks for clarifying.